<launch>
    <arg name="model_name"/>
    <arg name="x_pos" default="0"/>
    <arg name="y_pos" default="0"/>
    <arg name="yaw" default="0"/>
    <arg name="sensor_record" default="false"/>

    <include file="$(find cwru_turtlebot)/launch/includes/turtlebot_common.launch.xml">
        <arg name="model_name" value="$(arg model_name)"/>
        <arg name="x_pos" value="$(arg x_pos)"/>
        <arg name="y_pos" value="$(arg y_pos)"/>
        <arg name="yaw" value="$(arg yaw)"/>
        <arg name="sensor_record" value="$(arg sensor_record)"/>
    </include>

    <!-- Main ROS node for turtlebot logic-->
    <node pkg="cwru_turtlebot" type="stationary_turtlebot.py" name="robot" output="screen"/>
</launch>
